### PR TITLE
Remove file system/security/util/URLString.cs

### PR DESCRIPTION
**Pull request by rahku**

### CODE REVIEW

**Comment by rahku** [comment]
Fixes issue #8732

**Comment by rahku** [comment]
PTAL @jkotas I have removed function AppDomainSetup.NormalizePath and replaced its usage with Path.GetFullPath.

**Comment by rahku** [comment]
PTAL @JeremyKuhne I have removed the changes that you did for this comment. As it is not applicable for coreclr as there is no quirking config files available in coreclr.
// Long path checks can be quirked, and as loading default quirks too early in the setup of an AppDomain is risky.

File: src/mscorlib/src/System/AppDomainSetup.cs [hunk 1]
```diff
@@ -213,230 +208,29 @@ public String ApplicationBase
...
-                    path = AppDomain.NormalizePath(path, fullCheck: true);
+                Value[(int)LoaderInformation.ApplicationBaseValue] = (value == null || value.Length == 0)?null:Path.GetFullPath(value);
             }
-
-            return path;
         }
-
+        
         public String ConfigurationFile
         {
```
Comment (jkotas) [inline src/mscorlib/src/System/AppDomainSetup.cs]: This method is not used anywhere. It can be deleted.

Comment (jkotas) [inline src/mscorlib/src/System/AppDomainSetup.cs]: Bonus points for deleting all other properties in AppDomainSetup that are not used anywhere :-)

**Comment by jkotas** [comment]
LGTM modulo comment.

**Pull request merged**